modules/flowable-ui-modeler/flowable-ui-modeler-logic/src/main/java/org/flowable/app/service/api/ModelService.java

```diff
@@ -51,7 +53,7 @@
     
     Map<String,byte[]> getDecisionTableDefinitionsForProcess(String modelId);
 
-    BpmnModel getBpmnModel(AbstractModel model, Map<String, Model> formMap, Map<String, Model> decisionTableMap);
+    BpmnModel getBpmnModel(AbstractModel model, Map<String, Model> formMap, Map<String, Model> decisionTableMap, boolean resolveFormReference);
```

Comment:
Should we be making breaking changes to the API of this class?  Wont that make it harder to upgrade flowable versions?   Shouldn't we keep the existing method and just have it call our new impl with boolean=false?